data/json/bionics.json

```diff
@@ -64,34 +73,61 @@
     "name": { "str": "Alloy Plating - Legs" },
-    "description": "The flesh on your legs has been surgically replaced by alloy plating.  Provides passive protection and can be used in conjunction with bionic martial arts.",
-    "occupied_bodyparts": [ [ "leg_l", 6 ], [ "leg_r", 6 ] ],
-    "bash_protec": [ [ "leg_l", 3 ], [ "leg_r", 3 ] ],
-    "cut_protec": [ [ "leg_l", 3 ], [ "leg_r", 3 ] ],
-    "bullet_protec": [ [ "leg_l", 3 ], [ "leg_r", 3 ] ],
-    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF" ]
+    "description": "The skin and flesh on your legs has been surgically replaced by alloy plating.  Provides passive physical and chemical protection and can be used in conjunction with bionic martial arts at the cost of flexibility.",
+    "occupied_bodyparts": [ [ "leg_l", 7 ], [ "leg_r", 7 ] ],
+    "bash_protec": [ [ "leg_l", 7 ], [ "leg_r", 7 ] ],
+    "cut_protec": [ [ "leg_l", 5 ], [ "leg_r", 5 ] ],
+    "bullet_protec": [ [ "leg_l", 5 ], [ "leg_r", 5 ] ],
+    "fire_protec": [ [ "leg_l", 3 ], [ "leg_r", 3 ] ],
+    "acid_protec": [ [ "leg_l", 3 ], [ "leg_r", 3 ] ],
+    "env_protec": [ [ "leg_l", 2 ], [ "leg_r", 2 ] ],
+    "encumbrance": [ [ "leg_l", 4 ], [ "leg_r", 4 ] ],
+    "wet_protection": [ { "part": "leg_l", "ignored": 4 }, { "part": "leg_r", "ignored": 4 } ],
+    "covered_bodyparts": [ "leg_l", "leg_r" ],
+    "no_bleed_chance": 0.25,
+    "no_bite_chance": 0.33,
+    "cant_uninstal_msg": "The Alloy Plating is part of %s body now.  Removing it would leave %s disabled.",
+    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "BIONIC_CANT_UNINSTAL", "BIONIC_CANT_COMPRESS" ]
   },
   {
     "id": "bio_armor_torso",
     "type": "bionic",
     "name": { "str": "Alloy Plating - torso" },
-    "description": "The flesh on your torso has been surgically replaced by alloy plating, protecting it from physical trauma.",
-    "occupied_bodyparts": [ [ "torso", 10 ] ],
-    "bash_protec": [ [ "torso", 3 ] ],
-    "cut_protec": [ [ "torso", 3 ] ],
-    "bullet_protec": [ [ "torso", 3 ] ],
-    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF" ]
+    "description": "The skin and flesh on your torso has been surgically replaced by alloy plating, protecting it from physical and chemical injuries at the cost of flexibility.",
+    "occupied_bodyparts": [ [ "torso", 11 ] ],
+    "bash_protec": [ [ "torso", 7 ] ],
+    "cut_protec": [ [ "torso", 5 ] ],
+    "bullet_protec": [ [ "torso", 5 ] ],
+    "fire_protec": [ [ "torso", 3 ] ],
+    "acid_protec": [ [ "torso", 3 ] ],
+    "env_protec": [ [ "torso", 2 ] ],
+    "encumbrance": [ [ "torso", 5 ] ],
+    "wet_protection": [ { "part": "torso", "ignored": 4 } ],
+    "covered_bodyparts": [ "torso" ],
+    "no_bleed_chance": 0.25,
+    "no_bite_chance": 0.33,
+    "cant_uninstal_msg": "The Alloy Plating is part of %s body now.  Removing it would leave %s disabled.",
+    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "BIONIC_CANT_UNINSTAL", "BIONIC_CANT_COMPRESS" ]
```

Comment:
```suggestion
    "cant_uninstall_msg": "The Alloy Plating is part of %s body now.  Removing it would leave %s disabled.",
    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "BIONIC_CANT_UNINSTALL", "BIONIC_CANT_COMPRESS" ]
```